### ISSUE TITLE
KL计算代码像是写错了？

### ISSUE THREAD

**Issue by jinxinglu** [lancopku/DynamicKD#1]
models/student.py 的285行，
`F.softmax(large_logits[large_kd_indices], dim=-1))`
是不是应该为
`F.softmax(large_logits[large_kd_indices] / self.temperature, dim=-1))` ?
其它几处也是这样的问题

**Comment by TobiasLee** [comment]
Hi，你是对的，这里应该是都要除以 temperature 的。
文章中的设置 temperature 都等于 1 ，所以对于结果应该没有影响。

**Comment by jinxinglu** [comment]
👌
还有想问个问题，文章中只给出了三种Dynamic Adoption方法的独立实验结果，有没有将这三种方法结合在一起的结果呢？
谢谢

**Comment by TobiasLee** [comment]
暂时没有噢，这也是我们觉得未来值得探索的一个方向。一个有趣的问题这三个方面的作用是否是独立的？个人的看法是可能不是的，三者是有相互重叠的部分，例如在训练早期多学习（相对）简单样本和训练早期多学习 representation alignment 可能有相同的作用，文章的 Discussions 里也讨论了一些其他的方向。如果你有进一步的发现，欢迎 share 给我们~
